### PR TITLE
Adds PER support to the claim loop

### PR DESCRIPTION
**Pull request by rposborne**
This adds PER or Claim Contact Information loop support. 

This also adds easier to understand readers for payer contacts. This does not modify the csv output.

### CODE REVIEW

File: tests/test_edi_835_parser.py [hunk 1]
```diff
@@ -48,6 +49,21 @@ def test_total_interests(sample_935_with_interests):
 	assert sum_interests(sample_935_with_interests) == round(Decimal(10.3), 2)
 
 
+def test_can_parse_edi_files_without_warnings(exhaustive_sample_path, caplog):
+	import edi_835_parser
+
+	with caplog.at_level(logging.INFO, logger='edi_835_parser'):
+		edi_835_parser.parse(exhaustive_sample_path)
+		assert 'Identifier: PER not handled in claim loop.' not in caplog.text
```
Comment (curtisim0) [inline tests/test_edi_835_parser.py:57]: ohhhhhhh interesting

File: edi_835_parser/loops/service.py [hunk 2]
```diff
@@ -143,7 +143,7 @@ def build(
 					return service, segment, segments
 
 				else:
-					message = f'Identifier: {identifier} not handled in service loop.'
+					message = f'Identifier: {identifier} not handled in service loop.  {segment}'
```
Comment (curtisim0) [inline edi_835_parser/loops/service.py:146]: noith, does that have a useful str?

**Review by curtisim0 — APPROVED** [review]
LGTM

File: edi_835_parser/loops/service.py (hunk 2 above)
Comment (rposborne) [reply edi_835_parser/loops/service.py:146]: It just helps us find the "weird" when seeing the failure.   It really helps test authoring IMHO.

**Pull request merged**